apps/server/src/controllers/join-tournament.controller.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  PROTO_JOIN_LOBBY_ACCEPT,
+  PROTO_JOIN_LOBBY_REQUEST,
+} from '@razor/constants';
+import {
+  InitialClientData,
+  InitialServerData,
+  Snapshot,
+  socketId,
+  TournamentId,
+} from '@razor/models';
+import { store } from '@razor/store';
+import { pick } from 'lodash';
+
+import { PubSubEvents } from '../models';
+import { ContextOutput, Logger, pubsub } from '../services';
+import { tokenPlayerMap } from '../stores';
+
+interface JoinLobbyRequestArgs {
+  socketId: socketId;
+  data: InitialClientData;
+  context: ContextOutput;
+}
+
+const logger = new Logger('create-tournament.controller');
+
+pubsub.subscribe(
+  PROTO_JOIN_LOBBY_REQUEST,
+  ({ socketId, data, context }: JoinLobbyRequestArgs) => {
+    const { playerName, roomId } = data;
+    const receivedTournamentId: TournamentId = `T:${roomId}`;
+    const playerId = store.dispatch.game.joinPlayer({
+      receivedTournamentId,
+      playerName,
+    });
+    if (!playerId) {
+      logger.error("Store didn't send a playerId", context);
+      return;
```

Comment:
is just ignoring the request when an error occurs intended ? or are we going to send an error response?

Reply:
Opened a separate issue for this https://github.com/the-ai-team/razor/issues/80